deno.json

```diff
@@ -23,5 +23,14 @@
       "src/html",
       "tests/testdata"
     ]
+  },
+  "imports": {
+    "@david/dax": "jsr:@david/dax@0.40.1",
+    "@deno/cache-dir": "jsr:@deno/cache-dir@^0.11.1",
+    "@deno/graph": "jsr:@deno/graph@^0.82.3",
+    "@std/assert": "jsr:@std/assert@^0.223.0",
+    "browserslist": "npm:browserslist@4.23.0",
+    "lightningcss": "npm:lightningcss@^1.28.2",
```

Comment:
Maybe this needs to be `1.26.0` to pass the CI check?

Or, alternatively, we can commit the new css files to avoid css diffs